server/system/types/user.go

```diff
@@ -37,8 +37,16 @@ type (
 	}
 
 	UserMeta struct {
-		// @todo remove, obsolete.
-		Avatar string `json:"avatar,omitempty"`
+		// User's profile avatar photo attachment ID
+		AvatarID uint64 `json:"avatarID,string"`
+
+		// User's profile avatar initials attachment ID
+		AvatarInitialsID uint64 `json:"AvatarInitialsID,string"`
```

Comment:
Don't you think this is just going to complicate things? Why not have it under AvatarID?
If you really want to differentiate between them, you've added a special kind for generated avatars already.